docs/pages/messages/missing-root-task-in-turbo-json.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Missing root task in turbo.json
+description: Missing root task in turbo.json error
+---
+
+# Missing root task in turbo.json
+
+## Why This Error Occurred
+
+We call the scripts defined in the monorepo's root `package.json` the root tasks. These tasks often call `turbo`
+themselves. For example:
+
+```json package.json
+{
+  "scripts": {
+    "build": "turbo run build"
+  }
+}
+```
+
+This creates a problem when we declare task dependencies with `^`. A quick recap, `^<task>` means all of the `<task>`
```

Comment:
I'd prefer to link to where we explain this in the documentation...Crossing my fingers, do we have that somewhere?

Reply:
Yeah we do, but imo it's not the greatest explanation: https://turbo.build/repo/docs/reference/configuration#dependson